website/content/en/status/report-2025-04-2025-06/foundation-sta.adoc

```diff
@@ -0,0 +1,104 @@
+=== Infrastructure Modernization
+
+Contact: Ed Maste <emaste@FreeBSD.org> +
+Contact: Alice Sowerby <alice@freebsdfoundation.org>
+
+The project started in Q3 of 2024 and was commissioned by the Sovereign Tech Agency with a budget of $745,000, to be spent over about one year.
+The main goals are to improve security tools for the base system, ports, and packages, update the project's infrastructure to speed up development, enhance build security, and make it easier for new developers to get started.
+
+==== Q2 update
+
+All five work packages are now in progress and will run until the end of December 2025, at which time the project will close.
+
+==== Work Package A: Technical Debt reduction
+
+The majority of the work in this work package is complete, with a small number of hours allocated each month to help support FreeBSD Project's Source Management team to embed their new processes to make bug management easier and more sustainable.
+The bug backlog dashboard link:https://grimoire.freebsd.org[https://grimoire.freebsd.org] remains available to help make the backlog easier to understand.
```

Comment:
We do need the square parenthesis `[]`, but there is no need to put the link there: the link is the default displayed value.

```suggestion
The bug backlog dashboard link:https://grimoire.freebsd.org[] remains available to help make the backlog easier to understand.
```